.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+.idea
```

Comment:
Esto entra más dentro de la categoría de "good to know", pero para cuando trabajes en proyectos más grandes, puedes agarrar un preset de .gitignore de [acá](https://github.com/github/gitignore) y agregarle más cositas a tu conveniencia.

Reply:
Va, le pegaré una checada a la pagina y ya la guardé para tenerlo de referencia en ese tipo de proyectos. Thanks : D